src/createServer.js

```diff
@@ -1,3 +1,36 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+const http = require('http');
+const { convertToCase } = require('./convertToCase/convertToCase');
+const { validation } = require('./validation');
+
+const createServer = () => {
+  const server = http.createServer((req, res) => {
+    res.setHeader('Content-Type', 'application/json');
+
+    const normalizeURL = new URL(req.url, `http://${req.headers.host}`);
+    const originalText = normalizeURL.pathname.slice(1);
+    const targetCase = normalizeURL.searchParams.get('toCase');
+    const errors = validation(originalText, targetCase);
+
+    if (errors.errors.length) {
+      req.statusCode = 404;
+      req.statusMessage = 'Bad request';
```

Comment:
i think we should respond with 400 status, not request

Reply:
agree